broker-cli/proto/broker.proto

```diff
@@ -309,8 +309,11 @@ service AdminService {
  */
 enum TimeInForce {
   GTC = 0; // Good-til-cancelled, an order that remains in force until cancelled by the user
-  FOK = 1; // Fill-or-kill, an order that can be completely filled or will not be executed
-  IOC = 2; // Immediate-or-cancel, an order that will have as much filled immediately as possible with the remainder cancelled
+  // FOK is not supported
```

Comment:
[nit] would put this not in the comments next to the proto.

You should also check that the documentation generation from this change makes sense.

Reply:
Unfortunately this doesn't look well when rendered.

![image](https://user-images.githubusercontent.com/11034691/58727505-650b8700-8399-11e9-8399-b4680e8c4153.png)

Keeping these uncommented and updating the description to say the time restrictions for `FOK` and `IOC` are not currently implemented. This seems the most intuitive for backwards compatibility on indices, but I'm open to other approaches.